### ISSUE TITLE
ArrayIndexOutOfBoundsException in GridScrollExtension.getColumnWidth

### ISSUE THREAD

**Issue by alexandru-prigoreanu** [TatuLund/grid-scroll-extension#14]
hello! I encountered an exception using the following code
```
    /**
     * resets columns widths to the widths calculated by Vaadin.
     */
    private void resetColumnWidths() {
        grid.getColumns().forEach(column -> {
            double columnWidth = gridScrollExtension.getColumnWidth(column.getId());
            if (columnWidth > 0) {
                column.setWidth(columnWidth);
            }
        });
    }
```

I have a grid and a gridScrollExtension.
when i use the resetColumnWidths method on a freshly created grid everything works as expected.
when i remove grid's columns and add other columns and invoke again the method resetColumnWidths the following ArrayIndexOutOfBoundsException might get raised

```
java.lang.ArrayIndexOutOfBoundsException: 17
	at org.vaadin.extension.gridscroll.GridScrollExtension.getColumnWidth(GridScrollExtension.java:275)
	at it.imc.vaadin.ui.utils.grid.DynaEditableGrid.lambda$resetColumnWidths$0(DynaEditableGrid.java:146)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at java.util.Collections$UnmodifiableCollection.forEach(Collections.java:1080)
	at it.imc.vaadin.ui.utils.grid.DynaEditableGrid.resetColumnWidths(DynaEditableGrid.java:145)
```

Do you have any suggestions on the matter? Am i using the api in a wrong way? If there is a bug and you pinpoint me where and how to solve it i can try to create a pull request.
thank you for your time!

**Comment by TatuLund** [comment]
There is clearly a bug in the extension. The above or NPE can happen. The happens if there is column matching the id. NPE can happen if you you call the method with null. Will fix.

**Comment by TatuLund** [comment]
Fixed in 2.5.0. -1.0 returned if column not found.

**Comment by alexandru-prigoreanu** [comment]
thank you!
